### DISCUSSION TITLE
✅ 🚀 Feature Request: Accordion closed by default option

### DISCUSSION THREAD

**Discussion by seanbajuice** [elementor/elementor#20103] [Feature Request]
I have several sites now where I am using JavaScript to close accordions by default. Would be great if this was an option to do this in Elementor without using custom javascript.

**Comment by nicholaszein** [comment]
Hello there @seanbajuice!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the new nested Accordion in Elementor v3.15! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
